buku

```diff
@@ -2555,13 +2555,19 @@ class BukuDb:
                 LOGERR(e)
                 return False
 
+            add_parent_folder_as_tag = False
+            use_nested_folder_structure = False
             if not tacit:
-                resp = input('Add parent folder names as tags? (y/n): ')
+                resp = input('Add parent folder name as tags? (type "a" to add all folder names in case of nested structure) (y/a/n): ')
```

Comment:
i need second hand opinion here

old method is simple y/n -> add or not add

but this time y/a is ambiguous

maybe something similar like this

```
parent folder as tags?
y: add single direct parent folder as tag
a: add all folder in case of nested structure
n: dont add parend folder tag
(y/a/[n])
```
this also add information that `n` will be used as default if user don't choose anything

Reply:
So far program only used oneliners here, that's why i was hesitant. How about?
```
Add bookmark's parent folder as tag?
s: add single, direct parent folder
a: add all parent folders of the bookmark
n: don't add parent folder as tag
(y/a/[n])
```

Reply:
i don't recommend to use replace `y` with `s` other than that the description look fine